examples/citi_bike/online_lp/citi_bike_lp.py

```diff
@@ -0,0 +1,187 @@
+import math
+from typing import List, Tuple
+
+import numpy as np
+from pulp import LpProblem, LpVariable, LpMaximize, lpSum, LpStatus, GLPK
+
+from maro.utils import DottableDict
+
+class LP():
+    def __init__(
+        self, num_station: int, num_neighbor: int,
+        station_capacity: List[int], station_neighbor_list: List[List[int]],
+        decision_interval: int, config: DottableDict
+    ):
+        """A simple Linear Programming formulation for solving the bike repositioning problem.
+
+        Args:
+            num_station (int): Number of stations in current topology.
+            num_neighbor (int): Number of neighbors that needed to consider when repositioning.
+            station_capacity (List[int]): The capacity of each station. Length: num_station.
+            station_neighbor_list (List[List[int]]): The neighbor station of each station. Size: (num_station, num_station).
```

Comment:
why not use dict?

Reply:
the 1st level index indicates the station id, the 2nd level index indicates the order of the neighbor. I think List is more suitable here.